projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="a-input" [class.has-map]="showMap" [formGroup]="leafletForm">
+    <label *ngIf="label" class="a-input__label" for="leaflet-search">{{ label }}</label>
+    <div class="a-input__search">
+        <div class="a-input__wrapper has-inline-addon">
+            <input
+                    id="leaflet-search"
+                    type="text"
+                    name="leaflet-search"
+                    [placeholder]="placeholder"
+                    formControlName="searchField"
+                    autocomplete="off"
+                    (blur)="didSearch = false"
+            />
+            <div class="inline-addon">
+                <span *ngIf="searching" class="a-spinner a-spinner--sm u-margin-right-xs"></span>
+                <button
+                        class="a-button a-button--tiny"
+                        type="button"
+                        (click)="emptyField()"
+                        [disabled]="inputHasValue"
+                >
+                    <span class="show-on-desktop">Leegmaken</span>
+                    <span class="show-on-mobile"><i class="fa fa-times" aria-hidden="true"></i></span>
+                </button>
+            </div>
+        </div>
+
+        <div class="m-flyout m-flyout--full" [class.is-open]="didSearch">
```

Comment:
is there a specific reason for not using the autocomplete control anymore?

Reply:
Had a few issues using the autocomplete in a previous project (styling related) decided to just use an input field and the flyout html from the core branding github repo.